docs/reference/eql/detect-threats-with-eql.asciidoc

```diff
@@ -36,32 +36,54 @@ events imitating a Squiblydoo attack. The data has been mapped to
 
 To get started:
 
+. Create an <<index-templates,index template>> with
+<<create-a-data-stream-template,data stream enabled>>:
++
+////
+[source,console]
+----
+DELETE /_data_stream/*
+DELETE /_index_template/*
+----
+// TEARDOWN
+////
++
+[source,console]
+----
+PUT /_index_template/my-data-stream-template
+{
+  "index_patterns": [ "my-data-stream*" ],
+  "data_stream": { },
+  "priority": 500
+}
+----
+
 . Download https://raw.githubusercontent.com/elastic/elasticsearch/{branch}/docs/src/test/resources/normalized-T1117-AtomicRed-regsvr32.json[`normalized-T1117-AtomicRed-regsvr32.json`].
 
-. Use the <<docs-bulk,bulk API>> to index the data:
+. Use the <<docs-bulk,bulk API>> to index the data to a matching stream:
 +
 [source,sh]
 ----
-curl -H "Content-Type: application/json" -XPOST "localhost:9200/my-index-000001/_bulk?pretty&refresh" --data-binary "@normalized-T1117-AtomicRed-regsvr32.json"
+curl -H "Content-Type: application/json" -XPOST "localhost:9200/my-data-stream/_bulk?pretty&refresh" --data-binary "@normalized-T1117-AtomicRed-regsvr32.json"
```

Comment:
Note for reviewers:

If testing this, be sure to use the updated [normalized-T1117-AtomicRed-regsvr32.json](https://github.com/elastic/elasticsearch/pull/70822/files#diff-af9f5ca8d51631a3615392eaf1e7763868b42f28916dd7011a75e8938c003ad2) file in this PR. The one in the preview will be outdated until this PR is merged.